.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+end_of_line = lf
+insert_final_newline = true
```

Comment:
It seems that the file breaks its own rule :)

Reply:
😂 I think this was because I did not have any formatter installed for `.editorconfig` files. Added one manually :p